manifests/plugin/remote_execution/script.pp

```diff
@@ -1,11 +1,13 @@
-# = Foreman Proxy Remote Execution SSH plugin
+# = Foreman Proxy Remote Execution Script plugin
 #
-# This class installs Remote Execution SSH support for Foreman proxy
+# This class installs Remote Execution Script support for Foreman proxy
 #
 # === Parameters:
 #
 # $mode::               Operation Mode of the plugin.
 #
+# === SSH parameters:
+#
```

Comment:
Could we do something about the order in which installer's help shows those sections? Currently in `--full-help` we get  `Advanced`, then `SSH`, then `Basic`. Maybe `Basic`, then `SSH`, then `Advanced` would be the expected reading order?

Reply:
I think that is controlled by Kafo unfortunately

Reply:
Nevermind then